src/components/styles/CompaniesList.css

```diff
@@ -0,0 +1,116 @@
+.companies-list {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  width: 100%;
+  list-style: none;
+  background-color: #add8e6;
+  margin-bottom: 20px;
+}
```

Comment:
* As stated in the requirements kindly stick to the font-family given in the design as yours is different 
![image](https://github.com/ernestkorir/data-API/assets/52497246/600ae20e-4d12-45ec-82a4-c6baf125301d)